.travis.yml

```diff
@@ -57,19 +57,14 @@ before_install:
 install:
   - "travis_retry pip install -r .travis-${REQUIREMENTS}-requirements.txt"
   - "travis_retry pip install -e .[${EXTRAS}]"
+  - "pip freeze"
+
+before_script:
+  - "docker --version"
+  - "docker-compose --version"
 
 script:
-  - "./run-tests.sh"
+- "./run-tests.sh"
 
 after_success:
   - coveralls
-
-deploy:
```

Comment:
This was removed but no GH action added. Adding it in a different commit @lnielsen